cmd/state/main.go

```diff
@@ -82,6 +84,7 @@ func main() {
 		}
 	}
 
+	analytics.WaitForAllEvents(time.Second * 1)
```

Comment:
nit: I suggest putting this at the end of the captain command runner wrap that calls SetDefault (just to centralize the logic).

Reply:
Also nit: 2 or 3 more seconds should cover most circumstances where there is a long request hanging, but should not feel too bad to users (it should rarely happen).